torch/special/__init__.py

```diff
@@ -504,3 +504,29 @@
 
 Alias for :func:`torch.round`.
 """)
+
+log_softmax = _add_docstr(_special.special_log_softmax,
+                          r"""
+Computes softmax followed by a logarithm.
```

Comment:
This is an alias for https://pytorch.org/docs/master/generated/torch.nn.functional.log_softmax.html?highlight=log_softmax#torch.nn.functional.log_softmax, right?

We should probably update the OpInfo to reflect that this operation is canonically `torch.nn.functional.log_softmax`, too.

Reply:
Should these docs just point to the docs for torch.nn.functional.log_softmax and say this is an alias for it? 

Or is that not the case because the signatures are different?

Reply:
Yeah difference in signature leads to different behaviour so.
```python
>>> import torch
>>> t = torch.ones(2,2)
>>> torch.log_softmax(t, 0, torch.float64)
tensor([[-0.6931, -0.6931],
        [-0.6931, -0.6931]], dtype=torch.float64)
>>> torch.nn.functional.log_softmax(t, 0, torch.float64)
tensor([[-0.6931, -0.6931],
        [-0.6931, -0.6931]]) # Not torch.float64
```

Reply:
That seems like a bug in nn.functional.log_softmax. Would you file an issue, @kshitij12345?

Reply:
Worth an issue?

Reply:
Good idea -- might as well